tests/security/cc/cc_audit_test_setup.pm

```diff
@@ -41,8 +41,13 @@ sub run {
     # Install audit packages
     zypper_call('in audit audit-audispd-plugins');
 
-    # Export MODE
-    assert_script_run("export MODE=$audit_test::mode");
+    # Workaround for restarting audit service
+    assert_script_run('sed -i \'/\[Unit\]/a\StartLimitIntervalSec=0\' /usr/lib/systemd/system/auditd.service');
```

Comment:
I wonder about the `\S` that is included here, the beginning of "\StartLimitIntervalSec". Looks like you are escaping the "S" character by mistake here?

Reply:
Hmm, I misunderstood this message provided by `man sed`: 
```
a \ 
    text   Append text, which has each embedded newline preceded by a backslash.
```
Strange, it also works when there is a `\`. I modified the code to remove it. Thanks for this hint.

```
susetest:~ # sed '/\[Unit\]/a\StartLimitIntervalSec=0' /usr/lib/systemd/system/auditd.service
[Unit]
StartLimitIntervalSec=0
Description=Security Auditing Service
DefaultDependencies=no

```